sathesh77.md

```diff
@@ -0,0 +1,39 @@
+---
+name: "@sathesh77"
+Project name: "Brain computer interface(BCI)"
+---
+
+## Summary
+The interface proposed uses EEG signal of brain. The signals are used for performing certain tasks based on the human thinking. The interface is predominantly designed to subordinate disabled people. The interface mainly focuses people with locomotive disability. Based on the reference points in EEG this interface imitates locomotive movements to guide disabled.
+## FUTURE SCOPE
+It is a miniature model for a voice controlled wheel chair( For differently abled people). They can easily controll the wheel chair though commands. In further development it can also function using neural commands(Direct impulse from the brain).
+
+## Excited to Build
+The motivation behind in developing this interface is to guide and support people with locomotive disability and make them realize the physical happiness.
+## Plan
+The development has four zones. In the first zone study of biological actions of human brain, various biological markers of human EEG and its association with locomotive movements. Later blueprint workout of the model. Final stage involves in procurement and assembling the equipments. Once it's done testing involves real time testing with biological samples
+
+## Steps
+As mentioned in the development steps has different zones. Amongst studying brain through EEG has been done and study marks are marked.
+With respect to wireless connectivity to work on human signals, the demonstration was done using gesture of human and connectivity was established using Bluetooth.
+
+
+## Existing Project
+
+As of now I developed a Bluetooth RC Car that will work using mobile android app. In later I going to create control this car using brain signals with EEG devices then I like to create brain controlled wheelchair
+
+RC Car -https://projecthub.arduino.cc/samanfern/c71cd04b-79fd-4d0a-8a4b-b1dacc2f7725
+
+
+Project slide - https://docs.google.com/presentation/d/1srDQjUqZN2WoOYOOrdiLi9RqGjx2FWwF/edit?usp=drivesdk&ouid=100454217469060400921&rtpof=true&sd=true
+
+## Budget
+
+| Product                               | Supplier/Link                                                                                  | Cost   |
+| --------------------------------------| -----------------------------------------------------------------------------------------------| ------ |
+| Arduino uno                           |               already own
+|  Bluetooth/IC/Motor/chassis /wire     |               already own                                                                      | $0     |
+| BCI electrode(EEG single)             | https://shop.openbci.com/products/openbci-eeg-headband-kit                                     | $299   |
```

Comment:
This only has the electrodes. You'll need a controller.

Reply:
Thanks for replying 
After reviewing your valuable suggestions and recommendation, I have looked into various literature and found following products are appropriate both in terms of its efficiency and budget.

BCI::
Supplier Link🔗
https://www.pantechsolutions.net/brainsense-bci

Raspberry Pi 4::
Supplier Link🔗
https://www.thingbits.in/products/raspberry-pi-4-model-b-8-gb-ram

memory card::
Supplier Link🔗
https://www.thingbits.in/products/sandisk-32gb-class-10-microsd-memory-card-with-noobs

Micro HDMI to Standard HDMI Cable for Raspberry Pi 4 - 2 meter::
Supplier Link🔗
https://www.thingbits.in/products/micro-hdmi-to-standard-hdmi-cable-for-raspberry-pi-4-2-meter

Raspberry Pi 4 Official 15W USB C Power Supply ::
Supplier Link🔗
https://www.thingbits.in/products/raspberry-pi-official-15w-usb-c-power-supply-for-pi-4

Reply:
This looks better. Can you update the file with the new items, their prices in USD, and the total?

I worry that the [one probe for EEG](https://www.neuroelectrics.com/blog/2014/12/18/single-channel-eeg/) may not be sufficient for your aim, but I will let you make that determination yourself. Since it doesn't appear to have any specialized electronics, you may want to opt for [more probes](https://www.aliexpress.us/item/3256803886075504.html?spm=a2g0o.productlist.main.3.2089700fBEjclI&algo_pvid=b8ce1081-aedd-4cc8-ae34-5aa784c1fa87&aem_p4p_detail=202301222339013667430195055000018512274&algo_exp_id=b8ce1081-aedd-4cc8-ae34-5aa784c1fa87-1&pdp_ext_f=%7B%22sku_id%22%3A%2212000027945523742%22%7D&pdp_npi=2%40dis%21USD%2125.0%2125.0%21%21%21%21%21%4021227a0f16744595412481112d06b3%2112000027945523742%21sea&curPageLogUid=sdTpxFOIQDdK&ad_pvid=202301222339013667430195055000018512274_2&ad_pvid=202301222339013667430195055000018512274_2). 

As an aside, it looks like [this device](https://www.crowdsupply.com/hackerbci/pieeg) would be very helpful for your project but it's still in its preorder or crowdfunding phase unfortunately. But perhaps something to look into in the future.

Reply:
The project aims at acquiring only motion signal markers of EEG. So  we thought it would be sufficient to go with the product mentioned.